core/templates/pages/release-coordinator-page/release-coordinator-page.component.html

```diff
@@ -129,5 +129,12 @@ <h3>Enable Promo Bar</h3>
     border-color: #00376d;
     outline: 1px solid #00376d;
   }
-
+  @media (max-width: 600px) {
```

Comment:
Why is this needed? How would this affect the UI?

Reply:
I was trying to make a flake fix in this PR, but, yes maybe better to do in a separate PR.
Removed this.